src/server/pbsd_init.c

```diff
@@ -1449,6 +1452,54 @@ pbsd_init(int type)
 #endif
 
 	hook_track_recov();
+	
+	buf = NULL;
+	buf_len = 0;
+	for (i=0; i<svr_totnodes; i++) {
+		if (pbsndlist[i]->nd_attr[(int)ND_ATR_MaintJobs].at_flags & ATR_VFLAG_SET && 
+		    pbsndlist[i]->nd_attr[(int)ND_ATR_MaintJobs].at_val.at_arst->as_usedptr > 0) {
```

Comment:
I considered storing that into a different pointer.  The reason I didn't was because I need at_flags as well.  Looking at the code again, every other use of pbsndlist[i]->nd_attr[(int)ND_ATR_MaintJobs] is the arst ptr.  I did as your suggested.